lib/apiservers/engine/backends/volume.go

```diff
@@ -15,32 +15,92 @@
 package vicbackends
 
 import (
+	"encoding/json"
 	"fmt"
 	"net/http"
+	"os"
+	"strconv"
 
 	derr "github.com/docker/docker/errors"
 
 	"github.com/docker/engine-api/types"
 	"github.com/vmware/vic/lib/apiservers/portlayer/client/storage"
+	"github.com/vmware/vic/lib/apiservers/portlayer/models"
+	"github.com/vmware/vic/lib/guest"
 	"github.com/vmware/vic/pkg/trace"
 )
 
+//NOTE: These might be moved to a utility package once there are multiple personalities
+const optsVolumeStoreKey string = "VolumeStore"
```

Comment:
They are I will take a look at the cli options and make them look more like docker.